geom/CMakeLists.txt

```diff
@@ -1,20 +1,20 @@
-add_subdirectory (geom)        # special CMakeLists.txt
-add_subdirectory (geombuilder) # special CMakeLists.txt
-add_subdirectory (geompainter) # special CMakeLists.txt
-if(gdml)
+if(geo)
+  add_subdirectory (geom)        # special CMakeLists.txt
+  add_subdirectory (geombuilder) # special CMakeLists.txt
+  add_subdirectory (geompainter) # special CMakeLists.txt
+  if(gdml)
   add_subdirectory(gdml)
   if(NOT gnuinstall)
-    install(DIRECTORY gdml/ DESTINATION geom/gdml
-            FILES_MATCHING PATTERN "*.py"
-            PATTERN "inc" EXCLUDE
-            PATTERN "src" EXCLUDE)
+     install(DIRECTORY gdml/ DESTINATION geom/gdml
```

Comment:
This is NFC, right? Just whitespace?

Reply:
Yes. I wanted to follow the rule of the 2 spaces to indent within "scopes". The result is not very clear when looking at the diff, I agree.